lib/JSON/Validator.pm

```diff
@@ -574,69 +573,101 @@ sub _validate {
 
 sub _validate_all_of {
   my ($self, $data, $path, $rules) = @_;
-  my $type = data_type $data, $rules;
-  my (@errors, @expected);
+  my (@errors);
```

Comment:
No need for parenthesis.

Reply:
I removed most of them but missed this one.